frontend/src/pages/AccessRequestPage.tsx

```diff
@@ -0,0 +1,228 @@
+import React, { useEffect } from 'react';
```

Comment:
This file is a merge of Devin's original access request page and Jonny's edit user page, with a little chaos from me added into the mix.  Thanks guys for your hard work which allowed me to get this working.